STAR-87

softacoder opened the pull request:
- Update `Header.jsx` with logo and background color
- Update `Map.jsx` with Find us text to the google map
- Update `Hero.jsx`with text correction

softacoder committed:
Commit STAR-87

- Update `Header.jsx` with logo and background color
- Update `Map.jsx` with Find us text to the google map
- Update `Hero.jsx`with text correction

jarrodruby commented on client/src/components/Header.jsx:
```diff
@@ -2,17 +2,26 @@ import { AppBar, Box, Typography } from "@mui/material";
+        {/* Add an image in the left corner */}
+        <Box>
+          <img
+            src="/images/codeyourfuture.png" // Update the path accordingly
+            alt="CodeYourFuture Logo"
+            height={50} // Adjust the height as needed
```
Can we remove these comments

softacoder replied:
done

jarrodruby commented on client/src/components/Map.jsx:
```diff
@@ -1,9 +1,103 @@
-import { useRef, useEffect } from "react";
+// import { useRef, useEffect } from "react";
+// import { Loader } from "@googlemaps/js-api-loader";
+// import { Box } from "@mui/material";
+
+// const Map = () => {
+//   const mapContainerRef = useRef(null);
+
+//   useEffect(() => {
+//     const loader = new Loader({
+//       apiKey: "AIzaSyCODlhg_73yAwYNpOcjCE3nQOWWy-xioJs",
+//       version: "weekly",
+//     });
+
+//     const initialLocation = {
+//       lat: 51.52205390886717,
+//       lng: -0.08467381645964438,
+//     };
```
Better to delete old code. We can get it back from source control

softacoder replied:
done

softacoder committed:
Commit delete question in list of question

- In `Question.jsx` Remove the deleted question from the state
- In `questionsRouter.ts` added deleteQuestion
- In `questionsController` added Convert questionId to number

softacoder commented:
Jarrod I  made the fault to push code in this branch STAR 87 the are part of the delete question branch STAR 16. I have now correct and put in that code into STAR 16 also PR that branch and all work. Do i still need to click Resolve conflicts to fix in this branch?

softacoder commented:
Jarrod I made the fault to push code in this branch STAR 87 the are part of the delete question branch STAR 16. I have now correct and put in that code into STAR 16 also PR that branch and all work. Do i still need to click Resolve conflicts to fix in this branch?
